WooCommerce/WooCommerceTests/ViewRelated/Products/Edit Product/EditableProductVariationModelTests.swift

```diff
@@ -0,0 +1,49 @@
+import XCTest
+@testable import WooCommerce
+@testable import Yosemite
+
+final class EditableProductVariationModelTests: XCTestCase {
+    func test_a_variation_with_any_attribute_has_name_that_consists_of_all_attributes() {
+        // Arrange
+        let allAttributes: [ProductAttribute] = [
+            ProductAttribute(attributeID: 0, name: "Brand", position: 1, visible: true, variation: true, options: ["Unknown", "House"]),
+            ProductAttribute(attributeID: 0, name: "Color", position: 0, visible: true, variation: true, options: ["Orange", "Green"])
+        ]
+        // The variation only has one attribute specified - Color.
+        let variationAttributes: [ProductVariationAttribute] = [
+            ProductVariationAttribute(id: 0, name: "Color", option: "Orange")
+        ]
+        let variation = MockProductVariation().productVariation().copy(attributes: variationAttributes)
+
+        // Action
+        let name = EditableProductVariationModel(productVariation: variation, allAttributes: allAttributes).name
+
+        // Assert
+        let expectedName = [
+            "Orange",
+            String.localizedStringWithFormat(EditableProductVariationModel.Localization.anyAttirubteFormat, "Brand")
```

Comment:
```suggestion
            String.localizedStringWithFormat(EditableProductVariationModel.Localization.anyAttributeFormat, "Brand")
```

Reply:
Thanks for catching this Paolo! updated here: https://github.com/woocommerce/woocommerce-ios/pull/2608/commits/796493551517b221555ddf8129a956426a194b26